build.gradle.kts

```diff
@@ -1,8 +1,8 @@
 buildscript {
-    val aboutLibsVersion = "10.7.0"
-    val agpVersion = "8.1.1"
-    val kotlinVersion = "1.9.0"
-    val kspVersion = "1.0.11"
+    val aboutLibsVersion by extra("10.7.0")
```

Comment:
What about this? Shouldn't we name a variable for the version? Not adding the version as the name

Reply:
We could, but we don't have to. One could do it for added clarity, if you want.

At the moment it can be used like this: 
https://github.com/ArnyminerZ/icsx5/pull/22/files/3ddfeebc8b6d29f29387724cef88094af24f3535#diff-8cff73265af19c059547b76aca8882cbaa3209291406f52df1dafbbc78e80c46R107

Reply:
I guess it would actually be better to do give it an explicit name, since one might overlook the usage and rename either declaration.